src/sentry/sentry_apps/api/endpoints/sentry_app_authorizations.py

```diff
@@ -73,19 +73,22 @@ def post(self, request: Request, installation) -> Response:
                     user=promote_request_api_user(request),
                 ).run()
             else:
-                return Response({"error": "Invalid grant_type"}, status=403)
-        except APIUnauthorized as e:
-            logger.warning(
-                e,
+                raise SentryAppIntegratorError(
+                    message="Invalid grant_type", status_code=status.HTTP_403_FORBIDDEN
```

Comment:
can you make it consistent whether you use `status_code=status.XYZ` or `status_code=403` as a number?